examples/v1beta1/ubuntu-kubelet-log-query.yaml

```diff
@@ -0,0 +1,49 @@
+# This example provisioner will provision instances using the Ubuntu EKS AMI
```

Comment:
(nit) When a YAML file has a `---` line, I put one at the start as well.

That helps people learn that `---` is a document start marker, not a separator.